nni/common/concrete_trace_utils/concrete_tracer.py

```diff
@@ -1371,6 +1380,18 @@ def f(x, y):
         fx.GraphModule: a Module created from the recorded operations from ``root``.
     """
     tracer = ConcreteTracer()
+
+    args = inspect.getfullargspec(root.forward).args[1:]
+    defaults = inspect.getfullargspec(root.forward).defaults
+    if isinstance(concrete_args, (tuple, list)):
+        concrete_args = (*concrete_args, *defaults[len(concrete_args) + len(defaults) - len(args):])
+    else:
+        kv_default = {k: v for k, v in zip(args[-len(defaults):], defaults)}
+        concrete_args = {
+            **concrete_args,
+            **{n: kv_default[n] for n in args if n not in concrete_args}
+        }
```

Comment:
better put this inside ``ConcreteTracer().trace()``?

Reply:
good suggestion